### PR TITLE
Devtools and improvements

### PR DESCRIPTION
**Pull request by andrew-buckley**
Adds support for opening dev tools for debugging the desktop application. Improves bundle size. Mov
es webpack config and electron config to ts and splits webpack into dev and prod.

### CODE REVIEW

File: public/electron.ts [hunk 1]
```diff
@@ -66,12 +73,26 @@ function createMenu() {
                 { role: 'minimize' },
                 { role: 'close' }
             ]
+        },
+        {
+            label: 'Help',
+            submenu: [
+                {
+                    accelerator: 'CommandOrControl+Shift+I',
```
Comment (YingXue) [inline public/electron.ts]: is it the default shortcut that people follow for electron apps?

File: src/app/devices/deviceContent/components/deviceContentNav.tsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ interface DeviceContentNavState {
 }
 
 export const NAV_LINK_ITEMS_PNP = ['interfaces', 'settings', 'properties', 'commands', 'events'];
-export const NAV_LINK_ITEMS_NONPNP = ['identity', 'twin', 'events'];
+export const NAV_LINK_ITEMS_NONPNP = ['identity', 'twin', 'events', 'methods'];
```
Comment (YingXue) [inline src/app/devices/deviceContent/components/deviceContentNav.tsx]: snapshot test of this component needs to be updated accordingly. But before we do so, I'd like to add the labels at least on the deviceMethod before we enable it in the route as user would see it in the next release if we enable it

**Comment by andrew-buckley** [comment]
> Thank you so much for making all these wonder changes. I'd like to learn how the devtools works tomorrow from you.

Sure, stop by anytime.

File: public/electron.ts (hunk 1 above)
Comment (andrew-buckley) [reply public/electron.ts]: It's the shortcut that VSCode uses.

File: src/app/devices/deviceContent/components/deviceContentNav.tsx (hunk 2 above)
Comment (andrew-buckley) [reply src/app/devices/deviceContent/components/deviceContentNav.tsx]: Yeah I'm fixing the tests now and going to revert the methods change. That shouldn't go in this PR.

**Comment by andrew-buckley** [comment]
#97

**Pull request merged**